jest.config.js

```diff
@@ -1,4 +1,9 @@
 module.exports = {
+  globals: {
+    'ts-jest': {
+      diagnostics: false,
+    }
+  },
```

Comment:
If we do

```ts
ngParser.parseBinding(astInput, ...NG_PARSE_SHARED_PARAMS)
```

we need this, but 

```ts
ngParser.parseBinding.apply(ngParser, array)
```

seems don't need.

https://stackoverflow.com/questions/52118911/what-to-do-to-use-spread-operator-in-typescript

Reply:
I guess you meant to use `const array: any = ...` to bypass those type checks, which I don't think it's a good idea.

And since you disable the `diagnostics` feature for all the tests, we should add `yarn tsc -p . --noEmit` to `.travis.yml` so that we could type-check those tests.

Reply:
https://github.com/ikatyang/angular-estree-parser/pull/151/commits/81e53ce6839a3228d12c005c4c71299256180364